### PR TITLE
Add "starter" and tournament decks

### PR DESCRIPTION
**Pull request by axxroytovu**

### CODE REVIEW

**Comment by taw** [comment]
I think this might crash right now due to duplicate card protection, especially 22 basics. I'll need to verify that it doesn't, or add a switch to turn it off for some sheets/decks.

**Comment by axxroytovu** [comment]
Having duplicate card protection only on for draft boosters seems appropriate. There is no duplicate protection for set/collector boosters either.

File: data/boosters/3ed-starter.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Data from https://www.trollandtoad.com/magic-the-gathering/magic-the-gathering-sealed-product/revised-starter-deck-mtg-/87558#:~:text=Revised%20contains%20306%20new%20cards,%2C%20Tundra%2C%20and%20Tropical%20Island.
+name: "Revised Starter Deck"
+pack:
+  common: 11
+  uncommon: 3
+  rare: 1
```
Comment (taw) [inline data/boosters/3ed-starter.yaml:6]: This looks like just regular booster.

Comment (axxroytovu) [reply data/boosters/3ed-starter.yaml]: Fixed

**Comment by taw** [comment]
I added `duplicates: true` flag that skips all duplicate protection and makes every pull completely independent of each other.  Obviously it can't be combined with `balanced`. It should hopefully just work now.

**Comment by taw** [comment]
For what it's worth color balancing on a 32-element common sheet won't really do anything, all it does is guarantees at least 1 of each color, and then bumps probabilities of non-monocolored cards a bit so probabilities match.

With 32 commons, that's not really meaningful. But it's not going to cause any problems in practice.

**Comment by axxroytovu** [comment]
I'll keep that in mind for future products.

**Pull request merged**